frog/imports/api/activities.js

```diff
@@ -99,6 +150,15 @@ Meteor.methods({
     Activities.remove({})
   },
 
+  'graph.flush.operator': (operatorId) => {
+    Operators.remove({ _id: operatorId })
+  },
+
+  'graph.flush.operators': (activityId) => {
+    Operators.remove({ 'from._id': activityId })
+    Operators.remove({ 'to._id': activityId })
```

Comment:
Ok, I see, so you will use the field 'from._id' to retrieve the corresponding activity and find its position, duration, etc. I would have found more logical to get the plane information from the activity instead of the operator (planes are in the definition of activities, not of operators). But I understand that you had to find work-arounds against our incomplete always-changing implementation ;)